scripts/asm-installer/install_asm

```diff
@@ -1371,7 +1371,8 @@ init_meshconfig() {
 ${IDENTITY}
 EOF
     info "Cluster is in the Hub of project ${ENVIRON_PROJECT_ID} with Membership ID ${HUB_MEMBERSHIP_ID}"
-    local POST_DATA; POST_DATA='{"workloadIdentityPools":["'${ENVIRON_PROJECT_ID}'.hub.id.goog"]}'
+    # Support both Hub Workload Identity Pool and GKE Workload Identity Pool.
```

Comment:
Can you make the comment a bit clearer that this call replaces the pools and doesn't just initialize it? Maybe

```
# :initialize replaces the existing pools, so we need to support both Hub and GKE Workload Identity Pools
```